index.bs

```diff
@@ -2547,17 +2547,17 @@ Proposed Changes</h4>
 	<dfn lt="proposed change">proposed changes</dfn>.
 
 <h4 id="contributor-license">
-Contributor License Grants</h4>
+License Grants from Non-Participants</h4>
 
-	When a contributor who is not already obligated under the Patent Policy
+	When a party who is not already obligated under the Patent Policy
```

Comment:
nit, s/obligated/obliged/ Or s/is not already obligated under/has not yet incurred disclosure obligations under/

Reply:
Judging by the definitions, I think we should keep the wording as-is:
https://www.bing.com/search?q=define+oblige
https://www.bing.com/search?q=define+obligate